suite-native/module-authorize-device/src/components/passphrase/PassphraseForm.tsx

```diff
@@ -150,6 +156,7 @@ export const PassphraseForm = ({ inputLabel, onFocus }: PassphraseFormProps) =>
                                         horizontalMargin={FORM_CARD_PADDING}
                                     />
                                     <EnterPassphraseOnTrezorButton />
+                                    {noPassphraseEnabled && <NoPassphraseButton />}
```

Comment:
It's just a workaround, right? I guess in the end we should not request passphrase for standard wallets and user will have to choose the wallet before processing the request. Another workaround for all devices except T1 is entering passphrase on Trezor (you can enter empty passphrase there)...

Reply:
I don't mind this workaround, it does not affect the regular flow, so we can merge it.